content/src/main/content/jcr_root/apps/apm/clientlibs/editor/js/apm-edit.js

```diff
@@ -199,6 +200,11 @@
           self.fileUpload();
         });
 
+        this.$saveAndCloseButton.click(function () {
+          self.fileUpload();
+          window.location.href = DASHBOARD_URL;
```

Comment:
DASHBOARD_URL has been removed in https://github.com/Cognifide/APM/pull/120/files